### PR TITLE
Fix PE-D Bugs

### PR DESCRIPTION
**Pull request by PearlynnT**
Tester A for issues #168 and #171 has not been able to give a response on how to replicate the error.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -573,44 +597,46 @@ Checks if a group fulfils the group requirements of the course.
 
 #### Expected Output(Success):
 If the group fulfils the group requirements,
-* Message: "Group fulfils the diversity requirements of CS2103T. Group GROUP_NUMBER"
+* Message: "Group GROUP_NUMBER
+
```
Comment (wnchan) [inline docs/UserGuide.md:601]: is this a extra blank line?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -573,44 +597,46 @@ Checks if a group fulfils the group requirements of the course.
...
+    You can enter the `help` command for more information on group requirements."
 
-  You can enter the `help` command for more information on group requirements."
+If the group has only 1 member,
+* Message: "Group GROUP_NUMBER
 
-If not every member in the group has a tutorial that matches the group's tutorial,
-* Message: "Not every group member's tutorial matches the group's tutorial. Group GROUP_NUMBER
+    Group has only one member.
+
```
Comment (wnchan) [inline docs/UserGuide.md:616]: here too

**Review by wnchan — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/CheckCommand.java [hunk 3]
```diff
@@ -62,71 +71,81 @@ public CommandResult execute(Model model) throws CommandException {
...
-        if (count != groupMembers.size()) {
-            return new CommandResult(
-                    String.format(MESSAGE_CHECK_GROUP_TUTORIAL_WARNING + MESSAGE_HELP, groupToCheck.getNumber()),
-                false, false, true, false);
+
+        if (isSuccess) {
+            return new CommandResult(String.format(GROUP_NUM + MESSAGE_CHECK_GROUP_SUCCESS, groupToCheck.getNumber()),
+                    false, false, true, false);
+        } else {
+            return new CommandResult(String.format(GROUP_NUM + message + MESSAGE_HELP, groupToCheck.getNumber()),
```
Comment (maj0-0) [inline src/main/java/seedu/address/logic/commands/CheckCommand.java:146]: Thanks for adding the updated message!

**Review by maj0-0 — COMMENTED** [review]
LGTM!

File: docs/UserGuide.md (hunk 1 above)
Comment (PearlynnT) [reply docs/UserGuide.md]: no, it isn't an extra blank line but i've added breaks to improve readability

Comment (wnchan) [reply docs/UserGuide.md]: can use <br> instead or it isnt possible?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -303,6 +303,14 @@ Edits an existing student in the system.
 * You can remove all the student's social media by typing `sm/` without
   specifying any social media after it.
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A student can edit to include multiple tutorial groups they are interested in. Add multiple tutorial groups with multiple `t/`.
```
Comment (neyapraveen) [inline docs/UserGuide.md:307]: maybe by repeating `t/` would sound better?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -303,6 +303,14 @@ Edits an existing student in the system.
...
+</div>
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A student can edit to have any number of social media links. Add multiple social media links with multiple `sm/`.
```
Comment (neyapraveen) [inline docs/UserGuide.md:311]: same goes for sm

**Review by neyapraveen — COMMENTED** [review]
LGTM. Some suggestions given, and some merge conflicts

**Pull request merged**